src/locale/dcngettext.c

```diff
@@ -81,7 +81,7 @@ char *bindtextdomain(const char *domainname, const char *dirname)
 	}
 
 	UNLOCK(lock);
-	
+
```

Comment:
Remove extra newline.

Reply:
Not a new line. It seems the spaces are removed. I think this is automatically done by my editor. Please let me know if I should restore the spaces.

Reply:
Yes, please restore spaces. We want to make diff'ing the converted code with an unconverted one easier in future.